bokeh/models/annotations.py

```diff
@@ -133,6 +136,16 @@ class Annotation(Renderer):
 
     level = Override(default="annotation")
 
+@abstract
+class DataAnnotation(Annotation):
+    ''' Base class for data annotation models.
```

Comment:
We could be explicit here and explain that a `DataAnnotation` means that it has a `source` property